presidio-analyzer/presidio_analyzer/nlp_engine/transformers_nlp_engine.py

```diff
@@ -1,154 +1,134 @@
 import logging
-from typing import Optional, Dict
+from typing import Optional, Dict, List
 
 import spacy
-from spacy.language import Language
 from spacy.tokens import Doc, Span
 
-from presidio_analyzer.nlp_engine import SpacyNlpEngine
-
-
 try:
-    import torch
```

Comment:
Are we still using PyTorch elsewhere? If not, consider removing from requirements and maybe the notice?

Reply:
Good catch